security/security_profiles_operator/spo-seccomp.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="spo-seccomp"]
+= Managing seccomp profiles
+include::_attributes/common-attributes.adoc[]
+:context: spo-seccomp
+
+toc::[]
+
+// Abstract content here. Great place to define the difference between seccomp and selinux. Can also redirect to understanding page as needed or save that for additional resources at the bottom of the assembly.
```

Comment:
I could use some reference information about seccomp and selinux, why one is used over the other, pros/cons, etc.

Reply:
Let me know how deep we should go, but here's a short answer:

- seccomp allows to list what syscalls the app is allowed to do. Examples of syscalls include "open", "read", "write" etc. But, once you allow a syscall, all its uses are allowed, so once you allow "open", the app can open any file it wants.
- seccomp is more "standard", meaning it's available on all Linuxes which makes it IMO more portable in case the customer wants to reuse profiles across different Kubernetes distributions
- SELinux on the other hand works on combinations of "permissions" that are allowed on "labels". Everything on the system has a label (files, ports, devices, ...). This makes the SELinux policies more powerful, e.g. you can say that the app can call open on a file labeled with `var_log_t` but not on a file labeled with `etc_t`. Or you can say that the app can bind to a port labeled with `http_port_t` (I made the label up) but no others. 
- As a disadvantage, the SELinux policies are more complex to write and maintain, (e.g. files can change labels) and much less compact in comparison to seccomp
- SELinux is also enabled by default on a handful of Linux distributions, most notably the RH family. (but also all Android phones run with SELinux enforcing AFAIK..)

Should I dig up some better comparison?

Reply:
Thanks - between this and the presentation I have enough information!